Disambiguate Pod Phase from kubectl status... (#46299)
kubernetes/website: content/en/docs/concepts/workloads/pods/pod-lifecycle.md

Value       | Description
:-----------|:-----------
`Pending`   | The Pod has been accepted by the Kubernetes cluster, but one or more of the containers has not been set up and made ready to run. This includes time a Pod spends waiting to be scheduled as well as the time spent downloading container images over the network.

> `Running`   | The Pod has been bound to a node, and all of the containers have been created. At least one container is still running, or is in the process of starting or restarting.    

Author: ```suggestion
`Running`   | The Pod has been bound to a node, and all of the containers have been created. At least one container is still running, or is in the process of starting or restarting.
```

`Unknown`   | For some reason the state of the Pod could not be obtained. This phase typically occurs due to an error in communicating with the node where the Pod should be running.

{{< note >}}

> When a pod is not able to stay running, it is shown as `CrashLoopBackoff` by some kubectl commands.

Author: ```suggestion
When a pod is not able to stay running, it may be shown as `CrashLoopBackoff` by some kubectl commands in its `Status`.
```

> This is, however, not one of the not a Pod phases, that is to stay this "Status" is not a phase....

Author: ```suggestion
Make sure not to confuse "Status", a display field for user intuition, with the pod "Phase", which is an explicit part of the Kubernetes data model. 
```

> Make sure not to confuse "Status", a display field for user intuition, with the pod "Phase", which is an explicit part of the Kubernetes data model. 

Reviewer: I might write:
```suggestion
Make sure not to confuse _Status_, a kubectl display field for user intuition, with the pod [phase](/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase).
Pod phase is an explicit part of the Kubernetes data model and of the
[Pod API](/docs/reference/kubernetes-api/workload-resources/pod-v1/).
```

```
  NAMESPACE               NAME               READY   STATUS             RESTARTS   AGE

>   alessandras-namespace   alessandras-pod    0/1     CrashLoopBackOff   200        2d9h 

Reviewer: Is this indentation right?

```

> {{< /note >}}
> 
> 
> {{< note >}}

Reviewer: ```suggestion

---

```

Reviewer: (this separator renders in the right color for a note callout, making it more obvious that a new callout is starting)
